ui/app/styles/helper-classes/other.scss

```diff
@@ -3,84 +3,60 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-/* ARG TODO WIP: 
-- remove duplicate and overriding classes
-- address hard coded colors and sizing
-- continue to trim down. Goal is to have no helper file, but each of these helper in their respective helper-classes/file.scss
-*/
+/* This helper includes styles relating to: box-shadow, border, transition, pointer, and grouped css in one helper.*/
 
-.is-invisible {
-  visibility: hidden;
-}
-.is-sideless {
-  box-shadow: 0 2px 0 -1px $grey-light, 0 -2px 0 -1px $grey-light;
-}
-.is-bottomless {
-  box-shadow: 0 -1px 0 0 $grey-light;
-}
+// box-shadow helpers
 .has-only-top-shadow {
   box-shadow: 0 -1px 0 -1px $grey-light, 0 -2px 0 -1px $grey-light;
 }
+
 .has-bottom-shadow {
   box-shadow: $box-shadow !important;
 }
-.is-borderless {
-  border: none !important;
+
+.is-bottomless {
+  box-shadow: 0 -1px 0 0 $grey-light;
 }
-.is-box-shadowless {
+
```

Comment:
here are the two duplicate helper classes. They both had the class styling of `box-shadow: none !important`.